app/controllers/objects_controller.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'csv'
```

Comment:
This line is unnecessary, it's require by Bundler.

Reply:
The tests fail without it:

```     NameError:
       uninitialized constant CSV```

Reply:
I see, CSV is part of the standard library, not a gem, thus it's not in Gemfile and not automatically required 👍